lib/handlers/authorize-handler.js

```diff
@@ -53,21 +50,18 @@ function AuthorizeHandler(options) {
     throw new InvalidArgumentError('Invalid argument: model does not implement `getClient()`');
   }
 
-  if (!options.model.saveAuthorizationCode) {
```

Comment:
Also remove the `if (!options.authorizationCodeLifetime) {}` since it is being validated in the response type?